api/v1beta1/disruption_webhook.go

```diff
@@ -211,3 +225,148 @@ func (r *Disruption) getMetricsTags() []string {
 
 	return tags
 }
+
+// initialSafetyNets runs the initial safety nets for any new disruption
+// returns a list of responses related to safety net catches if any safety net were caught and returns any errors when attempting to run the safety nets
+func (r *Disruption) initialSafetyNets() ([]string, error) {
+	responses := []string{}
+	// handle initial safety nets if safemode is enabled
+	if r.Spec.Unsafemode == nil || !r.Spec.Unsafemode.DisableAll {
```

Comment:
I'm not exactly sure which booleans need to change, but currently, when `unsafeMode` is nil, we will run both safety checks. We will then get a panic when we check `unsafeMode.Config` at line 272

Reply:
Nice catch, I've added a check on line 272, if you think there should be something in Unsafemode itself let me know.